frontend/src/components/FullWidthAlert.tsx

```diff
@@ -25,7 +25,7 @@ const getBGColor = (type: Props["type"]) => {
 
 const FullWidthAlert = ({ type, heading, children }: Props) => {
   return (
-    <div className={`${getBGColor(type)} desktop:position-sticky top-0 z-200`}>
```

Comment:
Moving sticky classes to the BetaAlert component. If we reuse FullWidthAlert, it prob won't need to be sticky.